src/lib/glocal/glocal.ts

```diff
@@ -83,13 +83,13 @@ function glocalMissingLocalToolOnDiskMessage({
   // const packageManager = await createPackageManager(undefined, { projectRoot })
   //todo instead of "try your command again" write out what that command actually was
   return stripIndent`
-    The global ${toolName} CLI you invoked could not hand off to your project-local one becuase it wasn't on disk.
+    The global ${toolName} CLI you invoked could not hand off to your project-local one because it wasn't on disk.
     
     This can happen for example after you have cloned a fresh copy of your project from a remote repository.
 
     Please install your dependencies and try your command again.
 
-    Location of the ${toolName} CLI you invoked: ${execLayout.thisProcessToolBin.realPath}
-    Location of your project-local ${toolName} CLI: ${execLayout.project!.toolBinPath}
+    Location of the ${toolName} CLI you invoked: ${execLayout.thisProcessScriptPath}
+    Location of your project-local ${toolName} CLI: ${execLayout.project!.toolPath}
```

Comment:
Do you think code would be more readable if we harmonized the terms here?

```
processToolPath
projectToolPath
```

🤔

Reply:
I renamed `thisProcessScriptPath` to `processToolPath`. Is it fine keeping `project.toolPath`? Not sure it'd be better to have `project.dir`, `project.nodeModulesDir` and `projectToolPath` separate

Reply:
I don't know why I was using realPath before. User feedback here should be from point of view of what they gave which is _not_ real path. Maybe real path could be listed as additional data, but it shouldn't be the primary thing here.

But anyways, we can iterate...

Reply:
About naming, I tried the following:

```ts
process.toolPath
project.toolPath
```